### PR TITLE
Update Capstone to 3.0.1

### PR DESCRIPTION
**Pull request by aquynh**
Update Capstone to our latest stable release 3.0.1: http://capstone-engine.org/Version-3.0.1.html
Thanks.

### CODE REVIEW

File: Library/Formula/capstone.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 class Capstone < Formula
   homepage "http://capstone-engine.org"
-  url "http://capstone-engine.org/download/3.0/capstone-3.0.tgz"
-  sha1 "26e591b8323ed3f6e92637d7ee953cb505687efa"
+  url "http://capstone-engine.org/download/3.0.1/capstone-3.0.1.tgz"
+  sha1 "7f206c853c6b8d05de22e919c2455ab661654093"
   revision 1
```
Comment (DomT4) [inline Library/Formula/capstone.rb]: Can kill the ` revision ` line with the new version :)

Comment (aquynh) [inline Library/Formula/capstone.rb]: meaning that line can be deleted?

please feel free to do that, or do i need to submit a new PR?

thanks.

Comment (DomT4) [inline Library/Formula/capstone.rb]: Yup. No need to submit a new PR. You can make the change locally and then rebase and push; If you don't have the time or aren't that familiar with Git happy to talk you through it, or don't worry about it too much if you don't have the time :).

**Comment by aquynh** [comment]
i close this and will submit a new PR now.

thanks.

**Comment by aquynh** [comment]
hopefully this new PR is fine now: https://github.com/Homebrew/homebrew/pull/36516

Dominyk, thanks a lot!

**Comment by DomT4** [comment]
That looks perfect. Thanks @aquynh!

**Pull request closed without merging**